install/apm/node/install.yml

```diff
@@ -8,19 +8,8 @@ target:
   destination: application
   os:
     - linux
-  context: |
-    ###### WARNING
-    Using the command will automatically stop and attempt to restart all instrumentable applications, with all the same context, during the installation process. Once instrumented, redeploying any application(s) will require you to re-run the command. If you would like to manually install, please see our Node standard installation.
-
-    ###### INFO
-    This installation supports instrumenting Node processes managed by PM2. If you are using a different process manager, or are not using one at all, please see our Node standard installation.
-
+    
 install:
-  mode: targetedInstall
-  destination:
-    recipeName: node-agent-installer
-
-fallback:
```

Comment:
this should still have a `fallback` to the docs. The other apm install plans that use a setup nerdlet as the primary install function this way as well. 

Here's the link: https://docs.newrelic.com/docs/apm/agents/nodejs-agent/getting-started/introduction-new-relic-nodejs/#install